README.md

```diff
@@ -6,5 +6,11 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+Rails exists as a backend. It is used to facilitate interaction between two people (for example, playing a game online). It can also store the state of a game if the user wants to come back to it later without having to start over. It also stores registration and login information.
```

Comment:
Specifically, Rails is a framework for building back-end applications. To that end, it provides template code to start from (as well as generators to make more template code) which can then be customized for any particular purpose.